elasticsearch/src/main/java/org/apache/calcite/adapter/elasticsearch/ElasticsearchSchemaFactory.java

```diff
@@ -183,6 +186,15 @@ public ElasticsearchSchemaFactory() {
     }
   }
 
+  protected static List<HttpHost> getSortedHost(List<HttpHost> hosts) {
```

Comment:
LGTM!
Nitpick: The results and the following test are described as sortedHosts, I prefer getSortedHosts for this method.I'm still OK if it remains still.

Reply:
Thanks for your suggestion